x-pack/packages/kbn-entities-schema/src/schema/entity.ts

```diff
@@ -25,8 +25,8 @@ export interface MetadataRecord {
 
 export interface EntityV2 {
   'entity.id': string;
-  'entity.last_seen_timestamp': string;
   'entity.type': string;
+  'entity.last_seen_timestamp'?: string;
   [metadata: string]: any;
```

Comment:
Does this now also need a display_name field?

Reply:
yes good catch